### PR TITLE
Hotfix: Profile didn't remove records from manifest during uninstallation

### PR DESCRIPTION
**Pull request by EugeneButusov**

### CODE REVIEW

File: Soomla/Assets/Plugins/Soomla/Core/Config/android/SoomlaManifestTools.cs [hunk 1]
```diff
@@ -101,14 +101,51 @@ public static void GenerateManifest()
...
 				
-
-				
 				_document.Save(fullPath);
 			}
 
 
 		}
 
+		public static void ClearManifest(string moduleId) {
```
Comment (borisspektor) [inline Soomla/Assets/Plugins/Soomla/Core/Config/android/SoomlaManifestTools.cs]: @Blinkq why did you add another ClearManifest method? do we need the one without params?

Comment (Blinkq) [inline Soomla/Assets/Plugins/Soomla/Core/Config/android/SoomlaManifestTools.cs]: I left one with params, coz, if I d use it in Remove Soomla, it's been like,
4 times opening Manifest, 4 times reading it, 4 times checking what module we got and what to clear, 4 times close. ( If 4 modules in project)
So i think this realization more clean for runtime.

Comment (borisspektor) [inline Soomla/Assets/Plugins/Soomla/Core/Config/android/SoomlaManifestTools.cs]: @Blinkq ok but are we using the one without params?
It just looks like a lot of duplicate code.
If we're not using the one without params please remove it.
If we are, please remove duplicate code.

**Comment by EugeneButusov** [comment]
@borisspektor, your feedback was processed.
